epee/include/storages/portable_storage_template_helper.h

```diff
@@ -35,6 +35,17 @@ namespace epee
 {
   namespace serialization
   {
+    //-----------------------------------------------------------------------------------------------------------
+    template<class t_struct>
+    bool load_t_from_json(t_struct& out, const std::string& json_buff)
```

Comment:
Is this used in react-native-fast-crypto?

Is `react-native-fast-crypto/native-libs/test/main.cpp` the only portable_storage user?
If yes, perhaps we can drop portable_storage altogether?